inspectit-oce-bootstrap/src/main/java/rocks/inspectit/oce/bootstrap/noop/NoopContext.java

```diff
@@ -0,0 +1,34 @@
+package rocks.inspectit.oce.bootstrap.noop;
+
+import rocks.inspectit.oce.bootstrap.context.IInspectitContext;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class NoopContext implements IInspectitContext {
+
+    public static final NoopContext INSTANCE = new NoopContext();
+
+    @Override
+    public void setData(String key, Object value) {
+
```

Comment:
* remove this line
* add a private constructor so we enforce the usage of the `INSTANCE`